Kajabi/sage-lib
# [SAGE-513]: Page Heading Updates

## Description
Brings page headings up to design spec and includes the following:

- Updates description text color in the page heading.
- Adjusts the spacing of elements in the page heading toolbar to match the design spec.
- Brings charcoal color palette up to spec.

## Testing in `sage-lib`

- Visit http://localhost:4000/pages/component/page_heading
- Verify page heading toolbar buttons have the correct spacing
- Verify page heading intro text has the correct color.

## Testing in `kajabi-products`
1. (**LOW**) Color and spacing adjustments within page heading component.
   - [ ] Offers Page, Email Campaigns, People, or any page using the page heading component.

## Related
https://github.com/Kajabi/sage-lib/issues/513
